### ISSUE TITLE
Spherophakia

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7639]
**Preferred term label:**
Spherophakia  -- delineate from Mikrospherophakia

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://pubmed.ncbi.nlm.nih.gov/31274573/  2/2
601552 

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
